recipes/default.rb

```diff
@@ -80,6 +80,7 @@
     owner node.sensu.admin_user
     group node.sensu.group
     mode 0640
+    sensitive true if Chef::Resource::ChefGem.instance_methods(false).include?(:sensitive)
```

Comment:
see https://github.com/opscode-cookbooks/aws/pull/110 and http://jtimberman.housepub.org/blog/2015/03/20/chef-gem-compile-time-compatibility/